huaweicloud/services/vpc/resource_huaweicloud_networking_secgroup_rule.go

```diff
@@ -102,11 +102,11 @@ func ResourceNetworkingSecGroupRule() *schema.Resource {
 				),
 			},
 			"remote_group_id": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				ForceNew:     true,
-				Computed:     true,
-				ExactlyOneOf: []string{"remote_address_group_id", "remote_ip_prefix"},
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      true,
+				Computed:      true,
+				ConflictsWith: []string{"remote_address_group_id", "remote_ip_prefix"},
```

Comment:
"remote_address_group_id" and "remote_ip_prefix" can't be specified at same time

Reply:
the API (v1 and v3) can raise an error if more than one remote_xxx specified, so we can delete **ConflictsWith**